### DISCUSSION TITLE
Start an Argdown User Group

### DISCUSSION THREAD

**Discussion by cknoll** [christianvoigt/argdown#431] [Ideas]
There are probably people around the world who are using argdown and want to share their questions, ideas and results. As far as I see, there does not yet exist any communication space to easily reach other people who are also interested in the usage of Argdown (this repo IMHO is more dedicated to Argdown development).

Thus, I propose to start a low barrier communication channel (like a gitter chat, mailing list, separate dedicated github repo with discussion area, ...) and announce as "community" on in the README.md and on https://argdown.org/ (e.g. on the landing page and in the guide). This announcement could also include a disclaimer that the communication channel is self organized community project and the Argdown maintainers do not take any responsibility for what happens there.

**Background:** I am part of a german-speaking group ("Konstrukitive Digitale Diskussionskultur", https://kddk.eu) which aims to foster constructive discussions the digital realm. Among other things we use Argdown to reconstruct some selected debates from social media (especially mastodon).

**Comment by akersj14** [comment]
hi @cknoll, do you know if there has there been any movement on this? I am doing some research for my CS grad class on Argument Graph Software User Interfaces and trying to get a grip on who the general users are, what they are trying to do with this and what they want it to be able to do.
